lib/tests/execution/windows/execution.cc

```diff
@@ -90,39 +90,61 @@ SCENARIO("executing commands with execution::execute") {
         });
         return variables;
     };
-     GIVEN("a command that succeeds") {
-         THEN("the output should be returned") {
-            auto result = execute("cmd.exe", { "/c", "type", normalize(LIBFACTER_TESTS_DIRECTORY "/fixtures/execution/ls/file3.txt") });
-            REQUIRE(result.first);
-            REQUIRE(result.second == "file3");
+    GIVEN("a command that succeeds") {
```

Comment:
We're getting lots of duplicate code between the windows and posix tests... someday we might look for a way to clean that up.